spec/support/webmock_stubs_helper.rb

```diff
@@ -25,6 +25,20 @@ def stub_current_user_requests
       .to_return(status: 401, body: "")
   end
 
+  def stub_user_show_requests(user_id)
+    stub_user_show_request(user_id, "valid-access-token")
+
+    stub_request(:get, "https://#{accounts_host}/api/v1/users/#{user_id}.json")
```

Comment:
can we separate this to two methods?
`stub_user_show_request` and `stub_user_show_request_with_invalid_token`